packages/smooth_app/lib/pages/preferences/user_preferences_dev_mode.dart

```diff
@@ -488,3 +473,94 @@ enum DevModeScanMode {
     throw Exception('Unknown index $index');
   }
 }
+
+class LanguagePickerSetting extends StatelessWidget {
```

Comment:
Maybe we could move this inside widget class ? Would be more clean

Reply:
Sure